frontend/lib/ui/privacy-info-modal.tsx

```diff
@@ -34,7 +36,10 @@ export function PrivacyInfoModal(): JSX.Element {
               have regarding your personal information. If you’d like to read{" "}
               more, please review our full{" "}
               <OutboundLink
-                href="https://www.justfix.nyc/privacy-policy"
+                href={
+                  "https://www.justfix.nyc/privacy-policy" +
+                  (props.isForNorentSite ? "-norent" : "")
+                }
```

Comment:
Oh I just noticed that this is possibly a third place where the privacy policy link is hard-coded?  I think we should definitely move this out to a separate file and define it as a constant as part of our cleanup work!

Reply:
this is actually outdated!

Reply:
if you refresh, you should see that the url is saved as a variable here